jobs/integr8ly/tower-clean-uninstall.yaml

```diff
@@ -0,0 +1,113 @@
+---
+
+- job:
+    name: tower-clean-uninstall
+    description: 'Uninstalls Integreatly from fresh pure cluster created by Ansible Tower tooling where Integreatly has never been installed.'
+    project-type: pipeline
+    sandbox: false
+    concurrent: true
```

Comment:
Really? :)

Reply:
ah, true, there is hard-coded clusterName in the template so it means concurrent builds would step on each other toes.

Reply:
@tremes fixed